### DISCUSSION TITLE
Questions in regard to old react courses

### DISCUSSION THREAD

**Discussion by aScHEABL** [TheOdinProject/curriculum#25829] [General]
Could we please don't make the old react courses **gone**? 
Just move to somewhere else accessible **on the website** where we can still complete and then move onto the new courses, because I only have 2 lessons left from completing.
And I hate the feeling hanging there knowing there's something I haven't finished / worked on. 
Edit: Some of us also want the community solutions below the assignments back to the old courses.

**Comment by ManonLef** [comment]
@aScHEABL they will still be available in the [repository's archive](https://github.com/TheOdinProject/curriculum/tree/main/archive).

**Reply by rebsrebs** [reply]
I've just finished the CV App and was looking forward to comparing to other solutions but the solutions section seems to have disappeared today on the now "OLD" CV-App page. Are those solutions gone forever or are they archived somewhere? Could they be put back where they were?

**Reply by ManonLef** [reply]
I don't think it was very straightforward to preserve them with the new course but maybe another maintainer will chime in on that. 
In the meantime, if you're in our discord, [you can find many solutions in this creations showcase channel](https://discord.com/channels/505093832157691914/1013800461465309214)

**Reply by rebsrebs** [reply]
I think it's clear those are solutions specifically related to the instructions above rather than to the new course - you're not going to see them unless you are on that page marked OLD. I just spent 3 weeks working on this project and there are probably many others in the middle  or near the end of it - the solutions section is always a motivating end goal for me - when I finish a project I know I'll get to look through those and add my own -  so to have it disappear right at the finish line is pretty disheartening from a motivation standpoint.

**Reply by aScHEABL** [reply]
I second @rebeccode 's opinion, to be able to view community solutions is a big plus of TOP courses, it'll be a shame if we're unable to view it.
Sure we could still see the archived courses on the github but with community solutions **gone**.

**Reply by Eveieve** [reply]
I just found out that we can't view the old solutions in the CV project. Is there a particular reason as to why the maintainers decided to remove them? I get that there's no 'need' to view others' solutions - they're not answers, but it's just nice reading through others' code and seeing how they implemented things differently. We can view the new submissions but not the old ones on the website.

**Reply by ManonLef** [reply]
I spoke to the people that worked on this revamp. The old lesson will be gone in a week and so will the solutions with it. The idea really is to disregard the outdated workflow going forward since the new course includes the more current way of building React apps.

If you still feel strongly about viewing the solutions, Discord is really your best bet. You can search [the creations showcase channel on Discord](https://discord.com/channels/505093832157691914/1013800461465309214) and search for the project name to see solutions that people have submitted in that channel.

**Reply by CouchofTomato** [reply]
I believe the existing solutions from the old projects will be transferred to the new projects of the same title but it requires a little bit of work on our side which will be done shortly.

**Reply by Eveieve** [reply]
That's great to hear, thanks for following up on this. If there's anything that can be worked by a non-maintainer on this matter, I'm up for it.

**Reply by CouchofTomato** [reply]
Thanks for the offer. I am not sure who is handling it but believe the delay is because the new react course was being rolled out to all users in phases and we need to wait until that's complete

**Reply by 01zulfi** [reply]
A correction for the above comments: Solutions for the old projects won't be transferred over to the new projects.

This decision came to be for a few reasons. The most important one being that the old course and the new course are vastly different in their content and concepts covered. It is to be expected that the learners would approach the, let's say CV Application project differently if they had gone through the old content vs the new course.

**Reply by Eveieve** [reply]
I understand the maintainers' reasoning on this, thanks for the further explanation.

**Reply by aScHEABL** [reply]
I understand the reasoning but cannot wholeheartedly support this decision.
And there's more simple solution to this. Why not just make an archived section on the website directly and move the old and deprecated courses there so people like us who are interested just make ourselves there.
What's the reason for not doing it?
